### PR TITLE
Service borgs can now cook and miner borgs get a plasma cutter

### PR DESCRIPTION
**Pull request by thelaughingbomb**
## About The Pull Request

This PR brings in two new upgrade modules behind some research.

Service borgs can now get a rolling pin, knife and a item that manipulates food items.

Mining borgs get a plasma cutter that cannot weld and draws 250 charge from their cell per shot.

## Why It's Good For The Game

Service borgs are no longer just a drink dispenser on legs, they can actually make food now.
Mining as a borg is more bearable now since you have a plasma cutter.

## Changelog
:cl:
add: Cooking upgrade for service borgs
add: Plasma cutter for mining borgs
/:cl:

### CODE REVIEW

**Comment by thelaughingbomb** [comment]
As far as testing goes I researched and built the modules, installed and uninstalled them fine. I was able to make custom sandwiches by smacking food items together.

**Comment by coolsurf6** [comment]
SERVICE BORG CAN HANDLE BREAD!

oh seriously check if the sm bread will delete your tool and not the borg itself. I have a feeling it will.

**Comment by thelaughingbomb** [comment]
@coolsurf6 Yes, ive all ready tried holding anti bread, you can and it breaks the food manipulator. making it useless

**Comment by TheFakeElon** [comment]
> @coolsurf6 Yes, ive all ready tried holding anti bread, you can and it breaks the food manipulator. making it useless

does it stay broken after you put it away and grab it again? If it doesn’t just make it delete the manipulator or dust the borg

**Comment by coolsurf6** [comment]
> > @coolsurf6 Yes, ive all ready tried holding anti bread, you can and it breaks the food manipulator. making it useless
> 
> does it stay broken after you put it away and grab it again? If it doesn’t just make it delete the manipulator or dust the borg

It deletes the manipulator from what i was told in DMs

**Comment by thelaughingbomb** [comment]
@TheFakeElon the slot the manipulator was in becomes unusable

**Comment by thelaughingbomb** [comment]
conflict resolved, maybe?

File: austation/code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	storable = list(/obj/item/reagent_containers/food/snacks,
+	/obj/item/reagent_containers/food/condiment/flour,
+	/obj/item/reagent_containers/food/condiment/rice,
+	/obj/item/reagent_containers/food/condiment/soymilk,
+	/obj/item/reagent_containers/food/condiment/milk,
+	/obj/item/reagent_containers/food/condiment/mayonnaise,
+	/obj/item/reagent_containers/food/condiment/soysauce,
+	/obj/item/reagent_containers/food/condiment/sugar,
+	/obj/item/reagent_containers/food/condiment/peppermill,
+	/obj/item/reagent_containers/food/condiment/saltshaker) //yes this means you can do bread and i had to exempt enzymes so you dont pickup your internal one, which causes problems.
```
Comment (MCterra10) [inline austation/code/game/objects/items/robot/robot_items.dm:14]: you know you could just check if the object you're trying to grab is within your own body, right?

File: austation/code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -3,6 +3,11 @@
 /datum/techweb_node/unregulated_bluespace
 	design_ids = list("desynchronizer", "bluespace_jar")
 
+
+//Cyborg upgrades
+
+/datum/techweb_node/cyborg_upg_util
+	design_ids = list("borg_upgrade_holding", "borg_upgrade_lavaproof", "borg_upgrade_thrusters", "borg_upgrade_selfrepair", "borg_upgrade_expand", "borg_upgrade_rped", "borg_upgrade_circuitapp", "borg_upgrade_cooking", "borg_upgrade_cutter")
```
Comment (MCterra10) [inline austation/code/modules/research/techweb/all_nodes.dm:10]: don't override entire nodes, just add it as a comment.
I'm going to make a modularized way to add shit soon.

**Review by MCterra10 — CHANGES_REQUESTED** [review]

File: austation/code/game/objects/items/robot/robot_items.dm (hunk 1 above)
Comment (thelaughingbomb) [reply austation/code/game/objects/items/robot/robot_items.dm:14]: I wouldnt know how to do that

File: austation/code/modules/research/techweb/all_nodes.dm (hunk 2 above)
Comment (thelaughingbomb) [reply austation/code/modules/research/techweb/all_nodes.dm:10]: aight so you want me to move it to base code?

**Comment by ZombiesVsPlants** [comment]
bot angry

**Review by TheFakeElon — APPROVED** [review]

**Pull request merged**